### DISCUSSION TITLE
Capytaine v2.1

### DISCUSSION THREAD

**Discussion by mancellin** [capytaine/capytaine#494] [General]
Version 2.1 includes the following major changes:
- New feature: approximate forward speed, based on the work of [Donatini et al., 2022](https://www.researchgate.net/profile/Luca-Donatini/publication/361416824_Implementation_of_Forward_Speed_Effects_on_an_Open_Source_Seakeeping_Solver/links/62b03e41e1193368baac7b30/Implementation-of-Forward-Speed-Effects-on-an-Open-Source-Seakeeping-Solver.pdf).
- Nicer terminal displays using the [rich](https://rich.readthedocs.io) library (new mandatory dependency).
- More accurate tabulation of the Green function. The tabulation is now saved on disk.
- Optional new direct Boundary Integral Equation ("potential" formulation) instead of the default indirect BIE ("source" formulation).

This release also includes several bug fixes and minor usability improvements. See `docs/changelog.rst` for details.
It is also the first release to be compatible with Numpy 2.0.

<hr /><em>This discussion was created from the release <a href='https://github.com/capytaine/capytaine/releases/tag/v2.1'>Capytaine v2.1</a>.</em>

**Comment by baptistelabat-syroco** [comment]
Hi, I have tested the forward speed, and I was wondering if somebody knew if animation was able to take it into account.

**Reply by mancellin** [reply]
Well, there is this one: https://www.youtube.com/watch?v=_EPTrJXbGlY

But otherwise I never tried to update the animation examples in the documentation. I'm afraid the result would be disappointing: it would look the same except for some Doppler effect of the waves. Don't expect any kind of wake, the model is too simple for these.
